aten/src/ATen/cuda/detail/CUDAHooks.cpp

```diff
@@ -241,6 +241,9 @@ DeviceIndex current_device() {
   return -1;
 }
 
+/**
+ * DEPRECATED: use getCurrentDevice() instead
+ */
 DeviceIndex CUDAHooks::current_device() const {
   return at::cuda::detail::current_device();
 }
```

Comment:
@albanD Thanks for your approval. Should I delete these two deprecated APIs `current_device()` and `getNumGPUs()`?

Reply:
They are technically part of our public API, we would need to check on github in more depth to make sure they're not used before doing so.